src/web/components/ApiKeyManagement/KeyEditDialog.spec.tsx

```diff
@@ -0,0 +1,88 @@
+import { getAllowedRoles } from './KeyEditDialog';
+
+describe('test GetAllowedRoles function', () => {
```

Comment:
I think it's a bit misleading to have this file name and extension because it implies it is testing the dialog itself but it's really just testing one helper method.

Perhaps it would be better to extract `getAllowedRoles` to a `KeyEditDialogHelpers.ts` and then have this file be `KeyEditDialogHelpers.spec.ts` with the describe line being:
```
describe('#getAllowedRoles', () => {
```
This uses the convention we have in other files e.g. `siteConvertingHelpers.spec.ts` and `bulkAddPermissionsHelpers.spec.ts`

Reply:
Ah yeah that makes sense.